test/jdk/java/nio/channels/FileChannel/LoopingTruncate.java

```diff
@@ -48,7 +50,8 @@ public class LoopingTruncate {
     static long TIMEOUT = adjustTimeout(20_000);
 
     public static void main(String[] args) throws Throwable {
-        Path path = Files.createTempFile("LoopingTruncate.tmp", null);
+        Path dir = Path.of(StaticProperty.userDir());
```

Comment:
If you change this to Path.of(".") then it would avoid using a JDK internal class, Path.of("user.dir") will work too.

Reply:
If that approach is taken, then line 27 `@modules ...` can be removed.

Reply:
Switch to `Path.of(System.getProperty("user.dir"))` and documented the rationale in 8fc22379f774331be9a0dfea9a067483707c48df.